### PR TITLE
QA observation bugfix/FOUR-12799: Select lists are different controls in the launch pad modal

### PR DESCRIPTION
**Pull request by CarliPinell**
## How to Test
-  Open Any Process with "Open in Launchpad"
- Check both Dropdowns/Select List sizes

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-12799

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

File: resources/js/components/shared/IconDropdown.vue [hunk 1]
```diff
@@ -102,7 +102,9 @@ export default {
     onSearch(query) {
       this.query = query.toLowerCase();
       if (this.query.length) {
-        this.list = this.all.filter((icon) => icon.label.toLowerCase().includes(this.query));
+        this.list = this.all.filter((icon) =>
+          icon.label.toLowerCase().includes(this.query)
+        );
```
Comment (fagubla) [inline resources/js/components/shared/IconDropdown.vue:107]: keep it in a single line

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-b715487f89.engk8s.processmaker.net

File: resources/js/components/shared/IconDropdown.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/components/shared/IconDropdown.vue:107]: Ok. solved

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-b715487f89.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@CarliPinell 
Please review the height is not the same...
Please add a place holder in the icon to Select Icon
image

**Review by ana-mauricio — CHANGES_REQUESTED** [review]
HI @CarliPinell The text for icon 
image

It is not well centered, it seems to be at the top unlike the other select list
@ana-mauricio Last observation was fixed.

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-b715487f89.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
Carlos please replace Icon to Select Icon
image

**Review by ana-mauricio — APPROVED** [review]

**Comment by ryancooley** [comment]
> Carlos please replace Icon to Select Icon image

@CarliPinell Has this observation been addressed?

cc: @pmPaulis

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-b715487f89.engk8s.processmaker.net

**Pull request merged**